### PR TITLE
Version changes for v5.23.0

### PR DESCRIPTION
**Pull request by LiamConnors**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [5.23.0] - TBD
+## [5.23.0] - 2024-07-23
```
Comment (archmoj) [inline CHANGELOG.md:5]: This would be out today or tomorrow?

**Review by archmoj — APPROVED** [review]
:dancer:

File: CHANGELOG.md (hunk 1 above)
Comment (LiamConnors) [reply CHANGELOG.md:5]: Tomorrow

Comment (archmoj) [reply CHANGELOG.md:5]: Thank you :pray:

**Pull request merged**